rollup.config.js

```diff
@@ -6,13 +6,16 @@ import dts from 'rollup-plugin-dts';
 // We use esbuild for building the actual code.
 
 export default {
-  input: 'out/.dts/src/mod.d.ts',
+  input: 'out/.dts/mod.d.ts',
```

Comment:
argh...

The output uses the shortest common sub path and now that perf/ is no longer part of the project that changed.